src/main/java/io/opentracing/contrib/jdbc/TracingDriver.java

```diff
@@ -56,14 +75,105 @@ public Connection connect(String url, Properties info) throws SQLException {
     String realUrl = extractRealUrl(url);
     String dbType = extractDbType(realUrl);
     String dbUser = info.getProperty("user");
+    
+    Driver wrappedDriver = findTheDriver(realUrl);
 
-    // find the real driver for the URL
-    Driver wrappedDriver = findDriver(realUrl);
     Connection connection = wrappedDriver.connect(realUrl, info);
 
     return new TracingConnection(connection, dbType, dbUser, url.contains(WITH_ACTIVE_SPAN_ONLY));
   }
 
+  /**
+   *
+   * @param realUrl
+   * @return
+   * @throws SQLException
+     */
+  public Driver findTheDriver(String realUrl) throws SQLException {
```

Comment:
should be `private`

Reply:
please rename it as we have `findDriver` method. What is the difference? Please add also javadoc.